src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -140,6 +140,8 @@ internal override FlowAnalysisAnnotations FlowAnalysisAnnotations
                 return DecodeFlowAnalysisAttributes(GetDecodedWellKnownAttributeData());
             }
         }
+        public override bool IsNullChecked
```

Comment:
The change to public doesn't appear to affect the effective visibility (the containing class is `internal`.) Is this intended to become public API eventually?

Reply:
I'd gotten a build error at first, but it doesn't seem to be the case anymore. Will try reverting and pushing